aspnetcore/tutorials/signalr/includes/signalr3-5.md

```diff
@@ -0,0 +1,225 @@
+:::moniker range=">= aspnetcore-3.1 < aspnetcore-6.0"
```

Comment:
I believe we are supposed to put all prior versions in one include file.

Reply:
Thanks Tom.  Looking at the example provided at the kickoff of file versioning, the organization was this:
https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/fundamentals/configuration

- Latest version (7.0 in this case)
  - include/<6.0 versioned file>
  - include/<3-5 version file>

So everything below 5 is grouped, all other older versions get their own file in an include?
Did the plan change maybe and I'm not up on the latest or am I maybe misunderstanding the plan?

Reply:
@Rick-Anderson Should we put all prior versions in one include?